src/popup/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -363,6 +372,10 @@ export default defineComponent({
 
     const app = computed(() => popupProps.value?.app);
 
+    const isUnknownDapp = computed(() => (
+      !popupProps.value?.app || popupProps.value.app.name === UNKNOWN_SOURCE
+    ));
```

Comment:
Similar blocks of code is present in few places. Maybe this should be returned from `popupProps` composable?